src/init/features/apphosting.ts

```diff
@@ -31,6 +34,26 @@ export async function doSetup(setup: Setup, config: Config): Promise<void> {
       "Firebase App Hosting requires billing to be enabled on your project. Please enable billing by following the steps at https://cloud.google.com/billing/docs/how-to/modify-project",
     );
   }
+  await ensureApiEnabled({ projectId });
+  await ensureRequiredApisEnabled(projectId);
+  // N.B. Deploying a backend from source requires the App Hosting compute service
```

Comment:
Are we actually attaching the the storage.objectViewer role right now?
Also, even if we were doing that, this comment is about the storage.objectViewer role, but its on the block of code that actually provisions the SA, and we'd need to do that even if the use isn't using deploy from source

Reply:
Yeah, the storage.objectViewer role change isn't relevant until the follow-up deploy from source PR https://github.com/firebase/firebase-tools/pull/8516. Removing the comment.